theme/static/css/base.css

```diff
@@ -203,6 +232,30 @@ a:visited
   padding: 20px 0 0 676px;
 }
 
+.download-button {
+    display: flex;
+    padding: 14px 24px;
+    gap: 16px;
+    font-size: 24px;
+    color: #fff !important;
```

Comment:
I was always considering `!important` as hacks ;P
but non-blocking ;)

Reply:
Haha me too... and i would not have coded that if i had a fresh/blank start 😅